DefensiveItem.cpp

```diff
@@ -3,10 +3,11 @@
 #include "Paladin.h"
 #include "DragonSlayer.h"
 #include "Dragon.h"
-#include <cassert>
+#include "Utility.h"
```

Comment:
Don't include what you don't use
If you aren't using anything from these header files, don't include them
- [ ] revise